runtime/tests/runtime_integration_tests/test_ocp_lock/test_derive_mek.rs

```diff
@@ -0,0 +1,217 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_api::mailbox::{
+    CommandId, MailboxReq, MailboxReqHeader, OcpLockDeriveMekReq, OcpLockDeriveMekResp,
+};
+use caliptra_error::CaliptraError;
+use caliptra_hw_model::{HwModel, ModelError};
+
+use crate::test_ocp_lock::InitializeMekSecretParams;
+
+use super::{boot_ocp_lock_runtime, OcpLockBootParams};
+
+use zerocopy::{FromBytes, IntoBytes};
+
+/// Temporary checksum. This will be calculated from the full keyladder.
+const EXPECTED_MEK_CHECKSUM: [u8; 16] = [
+    243, 22, 202, 155, 137, 240, 133, 174, 210, 161, 117, 101, 142, 173, 20, 197,
+];
+
+// TODO(clundin): Follow up with the following test cases:
+//
+// These follow ups are pending an independently constructed key ladder / splitting into smaller
+// PRs.
+//
+// * Release MEK and validate MEK contents.
+// * Validate MEK checksum is correct.
+// * MEK and MEK checksum are the same after a warm reset.
+
+#[cfg_attr(not(feature = "fpga_subsystem"), ignore)]
+#[test]
+fn test_derive_mek() {
+    let mut model = boot_ocp_lock_runtime(OcpLockBootParams {
+        hek_available: true,
+        force_ocp_lock_en: true,
+        init_mek_secret_params: Some(InitializeMekSecretParams {
+            sek: [0xAB; 32],
+            dpk: [0xCD; 32],
+        }),
+        ..Default::default()
+    });
+
+    let mut cmd = MailboxReq::OcpLockDeriveMek(OcpLockDeriveMekReq {
+        hdr: MailboxReqHeader { chksum: 0 },
+        reserved: 0,
+        mek_checksum: [0; 16],
```

Comment:
Should this be the expected checksum? There is a test below with default value. Maybe I'm missing something.


```suggestion
        mek_checksum: EXPECTED_MEK_CHECKSUM,
```